activities/human-resources/job-description.md

```diff
@@ -0,0 +1,51 @@
+---
+type: Resources
+---
+
+# [Job title]
+
+Come help producers of open source software for public use – such as in cities and other public organizations – develop and maintain software and policy together. The Foundation for Public Code provides review of codebases that are in our stewardship to help developers build trusted and reliable software that can be widely reused and foster a strong community around it.
+
+We test against the Standard for Public Code to make codebases understandable, reusable and community based. Our standards are high, as well as opinionated, to ensure codebases can be trusted as the infrastructure of society and improve continually, and policy makers can keep technology and architectural choices open.
+
+Part of this effort is to [bespoke text contextualising role in broader foundation work].
+
+## What the [JOB TITLE] does
+
+As a [JOB TITLE] you work together with the community to [bespoke text]. This means you will:
+[bespoke text starting with a verb]
+
+[CALL TO ACTION - bespoke text e.g. So, if you enjoy working together with a wide array of developers to make open source codebases really shine as well as improve the role of technology in society, this job is for you.]
+
+## What we expect from all of our team members
+
+You should:
+* share our core values: openness, quality, trust, community, helpfulness
+* understand the value of and be experienced with open source products and communities
+* want to work for the public good
+* have a collaborative work attitude and take pride in making things together
+* be able to work independently
+* be a strong communicator in English
+* have international experience
+* be an active participant in the codebase community you support
+
+## What we expect from the [JOB TITLE]
+
+We expect our new [JOB TITLE] to:
+[bespoke text, starting with a verb]
+...
+
+## More information about this position
+
+This is a full time role (40 hours/week) in our main offices in Amsterdam, the Netherlands. Assistance in relocation is possible.
+
+This is a one year contract, with an option to extend. Compensation is competitive as well as in line with the public mission of the organization. 
+
+## More about the application process
+
+We enthusiastically encourage people underrepresented in the worlds of technology and government to apply.
+If you want to know more about the position, please get in touch with Boris van Hoytema at boris@publiccode.net or +31 6 17 96 02 05.
```

Comment:
Proposal: let's not leave Boris's phone number on the public internet forever in a template. Let's add it only when it's directly relevant. (Let's have an institutional bias towards privacy for people's personal data.)

```suggestion
If you want to know more about the position, please get in touch with Boris van Hoytema at boris@publiccode.net.
```

Reply:
I think this is a valid proposal, but feel this might not be the place, perhaps we buy a seperate VOIP number or sim card I can use for the Foundation for Public Code to receive calls on. Doesn't need to block merging I'd say.